python/ufoff.py

```diff
@@ -0,0 +1,114 @@
+
+from ufoLib2 import Font
+import flatbuffers
+from FlatFont.Ufo import FontInfo
+from enum import IntEnum
+
+SCALARS = [int, float, bool] # Other scalars needed # Enum needed
+NON_SCALARS = [str, list] # Other none-scalars needed
+
+def Capital(string: str): # seek equivalent buit-in function
```

Comment:
you mean https://docs.python.org/3/library/stdtypes.html#str.title ?

Reply:
No. I need a function to capitalize the first character and leave all other characters unchanged. "Str.title()" makes the remainder lowercase.